ext/tidy/tidy.c

```diff
@@ -826,8 +816,16 @@ static PHP_MINIT_FUNCTION(tidy)
 	tidySetPanicCall(php_tidy_panic);
 
 	REGISTER_INI_ENTRIES();
-	REGISTER_TIDY_CLASS(tidy, doc,	NULL, 0);
-	REGISTER_TIDY_CLASS(tidyNode, node,	NULL, ZEND_ACC_FINAL);
+
+	tidy_ce_doc = register_class_tidy();
+	tidy_ce_doc->create_object = tidy_object_new_doc;
+	memcpy(&tidy_object_handlers_doc, &std_object_handlers, sizeof(zend_object_handlers));
+	tidy_object_handlers_doc.clone_obj = NULL;
+
+	tidy_ce_node = register_class_tidyNode();
+	tidy_ce_node->create_object = tidy_object_new_doc;
```

Comment:
```suggestion
	tidy_ce_node->create_object = tidy_object_new_node;
```

Reply:
Ah, yes! I've amended my previous commit, but only changed this line.